src/scss/01_base/_base.scss

```diff
@@ -79,6 +79,20 @@ ul {
   padding: 0;
 }
 
+svg {
+  box-sizing: border-box;
+  display: inline-block;
+  fill: currentColor;
+  height: auto;
+  width: auto;
+
+  &[role=icon] {
+    height: 1em;
+    vertical-align: middle;
+    width: 1em;
+  }
+}
+
```

Comment:
Mi fa un po' paura questo... Sarebbe possibile evitare di metterlo in Pyxis?

Reply:
Aiuterebbe per dare uno standard a come si devono utilizzare gli SGV in caso siano icone. Per il resto se non si utilizza `role="icon"` non cambia nulla